### PR TITLE
Review model changes

### PR DESCRIPTION
**Pull request by jamiesprax**

### CODE REVIEW

**Comment by AdamDM** [comment]
lgtm

File: atlas-api/src/main/java/org/atlasapi/output/writers/ReviewsWriter.java [hunk 1]
```diff
@@ -38,9 +38,16 @@ public String fieldName(Review entity) {
     public void write(@Nonnull Review entity, @Nonnull FieldWriter writer, @Nonnull OutputContext ctxt)
             throws IOException {
 
-        String language = (null != entity.getLocale()) ? entity.getLocale().getLanguage() : null;
+        String language = (entity.getLocale() != null) ? entity.getLocale().getLanguage() : null;
+        String reviewType = (entity.getReviewType() != null) ? entity.getReviewType().toKey() : null;
+
         writer.writeField("language", language);
```
Comment (dragospitica) [inline atlas-api/src/main/java/org/atlasapi/output/writers/ReviewsWriter.java]: if the entity.getLocale() is null, will it not throw an NPE?

Comment (dragospitica) [inline atlas-api/src/main/java/org/atlasapi/output/writers/ReviewsWriter.java]: if `entity.getLocale()` is null, will it not throw an NPE?

File: atlas-api/src/main/java/org/atlasapi/output/writers/ReviewsWriter.java [hunk 2]
```diff
@@ -38,9 +38,16 @@ public String fieldName(Review entity) {
...
         writer.writeField("review", entity.getReview());
+        writer.writeField("author", entity.getAuthor());
+        writer.writeField("author_initials", entity.getAuthorInitials());
+        writer.writeField("rating", entity.getRating());
+        writer.writeField("date", entity.getDate());
+        writer.writeField("review_type", reviewType);
```
Comment (dragospitica) [inline atlas-api/src/main/java/org/atlasapi/output/writers/ReviewsWriter.java]: ditto ^

File: atlas-api/src/main/java/org/atlasapi/output/writers/ReviewsWriter.java (hunk 1 above)
Comment (jamiesprax) [inline atlas-api/src/main/java/org/atlasapi/output/writers/ReviewsWriter.java]: nope

File: atlas-api/src/main/java/org/atlasapi/output/writers/ReviewsWriter.java (hunk 2 above)
Comment (jamiesprax) [inline atlas-api/src/main/java/org/atlasapi/output/writers/ReviewsWriter.java]: ditto nope

**Pull request merged**